lonboard/_map.py

```diff
@@ -219,15 +211,15 @@ def on_click(self, callback: Callable, *, remove: bool = False) -> None:
     - Default: `5`
     """
 
-    basemap_style = BasemapUrl(CartoBasemap.PositronNoLabels)
-    """
-    A URL to a MapLibre-compatible basemap style.
+    basemap = t.Instance(MaplibreBasemap, allow_none=True).tag(
```

Comment:
Before merging we may keep `basemap_style` around to not break backwards compatibility, but I think documenting that the new API is
```py
Map(basemap=MaplibreBasemap(basemap_style=basemap_style))
```
isn't too bad